config/jobs/kube-oidc-proxy/kube-oidc-proxy-presubmits.yaml

```diff
@@ -210,7 +210,7 @@ presubmits:
         - e2e
         env:
         - name: KUBE_OIDC_PROXY_K8S_VERSION
-          value: "1.14.3"
+          value: "1.14.10"
```

Comment:
👍 we use `1.14` in cert-manager to avoid having to bump this here so often, but LGTM.